exchanges/poloniex/poloniex_types.go

```diff
@@ -1032,8 +1033,5 @@ func (s *V3ResponseWrapper) Error() error {
 	if s.Code != 0 && s.Code != 200 {
 		return fmt.Errorf("error code: %d; message: %s", s.Code, s.Message)
 	}
-	if s.Data == nil {
-		return common.ErrNoResponse
-	}
```

Comment:
🚧 Change request:
Please readd this.
You removed it because of the mock recorder.
Just update the http.json to give us an empty hash `{}`
See gbjk@97074e49e:
```
curl -s https://github.com/gbjk/gocryptotrader/commit/97074e49e.patch | git am
```